src/Compilers/VisualBasic/Portable/CodeGen/EmitExpression.vb

```diff
@@ -153,6 +156,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGen
             EmitPopIfUnused(used)
         End Sub
 
+        Private Sub EmitComplexConditionalAccessReceiver(expression As BoundComplexConditionalAccessReceiver, used As Boolean)
```

Comment:
This is not expressible as a ternary conditional expression because that expression never represents an LValue. In addition, CodeGenerator doesn't know how to load an address of a ternary conditional expression without cloning its value in a temp, which is not what we want.